### DISCUSSION TITLE
Inconsistent example sentence for 本心

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#15] [Errors and Corrections]
Japanese:

> 彼は口やかましいが本心は悪くない。

English:

> His bark is worse than his bite.

Suggested change:

> [EN] He's a grumbly person, but deep down he's not a bad guy

---

Translating 口やかましい is a bit tricky if we want to keep the same style but I think "grumbly" or "grumbly person" works well enough, and the reworked sentence shows the actual meaning of 本心 ("deep down") while keeping it natural-sounding in English. The original English sentence is way too metaphorical to be useful as a parallel with the Japanese in my opinion and doesn't really catch the real nuance of what the Japanese sentence is implying.

**Comment by stephenmk** [comment]
My spellcheck dictionary doesn't like "grumbly." I don't think it's necessarily bad to use informal language in example sentences where it's appropriate, but it's probably better to stick to standard vocabulary if it works just as well. Personally, depending on the context I'd probably go with "hard-ass" for this sentence, but that word might have too many meanings for a direct translation of 口やかましい. 

I linked the sentence to a new translation:

> He grumbles a lot, but deep down he's not a bad guy.

Thanks again for feedback.
